#roi info
limcheck/ImagePos roi

#Vision info
uint8 camera_num
uint8 dowel
limcheck/ImagePos[] glue